modules/db/Makefile

```diff
@@ -29,6 +29,6 @@ cloc-test:
 	cloc test
 
 clean:
-	rm -rf .clj-kondo/.cache .cpcache target
+	rm -rf .clj-kondo/.cache .cpcache target resources
```

Comment:
We have files in `resources` please find another way to clean only the profiles.